src/compiler/checker.ts

```diff
@@ -317,21 +317,6 @@ namespace ts {
         const emitResolver = createResolver();
         const nodeBuilder = createNodeBuilder();
 
-        const globals = createSymbolTable();

```

Comment:
NOTE: I needed to move these after the `const checker = {...}` declaration as `createSymbol` now depends on `checker` to ensure we add the current `TypeChecker` to all symbols to make sure we are always using the correct checker.